cvat/apps/documentation/user_guide.md

```diff
@@ -24,97 +24,155 @@ There you can:
 
-    ![](static/documentation/images/image006.jpg)
+__Segment size__. Use this option to divide a huge dataset into a few smaller segments.
 
-3. Follow a link inside ``Jobs`` section to start annotation process. In some cases you can have several links. It depends on size of your task and ``Overlap Size`` and ``Segment Size`` parameters. To improve UX only several first frames will be loaded and you will be able to annotate first images. Other frames will be loaded in background.
+__Image Quality__. Use this option to specify quality of uploaded images. The option helps to load high resolution datasets faster. Use the value from ``1`` (completely compressed images) to ``95`` (almost not compressed images).
+
+__Select files__. Push this button to select files you want to annotate.
+
+Push ``Submit`` button and it will be added into the list of annotation tasks. Then, the created task will be displayed on dashboard:
+
+![](static/documentation/images/image006.jpg)
+
+3. The Dashboard contains elements and each of them relates to a separate task. They are sorted in creation order. Each element contains: task name, preview, execution status, buttons, and one or more links. Each button is responsible for a specific function:
+- ``Dump Annotation`` — download an annotation file from the task (xml format) 
+- ``Upload Annotation`` — uploading an annotation file to the task (xml format)
+- ``Update Task`` — bring up "Update task" panel. It is used to edit or add labels line
+- ``Delete Task`` — delete the task
+- ``Git Repository Sync`` — sync annotation with the repository. Presence depends on task configuration
+- ``Run TF Annotation`` — automatic annotation with Tensorflow Object Detection API. Presence depends on task configuration
+- ``Run Auto Annotation`` — automatic annotation with  OpenVINO toolkit. Presense depends on how you build CVAT instance.
+
+  Item color depends on status of synchronization with the repository: ``red`` means a task is not synchronized with the repository, 
+``yellow`` means a task is in a temporary branch of the repository, ``green`` means a task is merged into the repository.
+
+4. Follow a link inside ``Jobs`` section to start annotation process. In some cases, you can have several links. It depends on size of your task and ``Overlap Size`` and ``Segment Size`` parameters. To improve UX, only the first several frames will be loaded and you will be able to annotate first images. Other frames will be loaded in background.
 
     ![](static/documentation/images/image007.jpg)
 
+
+### Creating a task using model manager
```

Comment:
"Creating a task using model manager"?

Reply:
Ok..Renamed this paragraph to "Model manager"